lib/config.js

```diff
@@ -69,6 +69,15 @@ const DEFAULTS = {
         monthlyBytes: 180000000000
       }
     }
+  },
+  redis: {
+    host: '',
+    port: '',
+    path: '',
```

Comment:
Need to remove all but host and port so it doesn't default to TLS and empty password, etc...